python/lsst/ts/vent/controller/controller.py

```diff
@@ -0,0 +1,314 @@
+# This file is part of ts_atbuilding_vents
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import os
+from enum import IntEnum
+
+import megaind
+import pymodbus.client
+
+from . import config as cfg
+from . import vfd
+
+__all__ = ["Controller"]
+logger = logging.getLogger(__name__)
+
+
+class VentGateState(IntEnum):
+    CLOSED = 0
+    """
+    The vent is closed
+    """
+
+    PARTIALLY_OPEN = 1
+    """
+    The vent is neither open nor closed
+    """
+
+    OPEN = 2
+    """
+    The vent is open
+    """
+
+    FAULT = -1
+    """
+    The vent is both open and closed
```

Comment:
This seems a bit weird no? Is this how you detect that there is a fault? I would suggest rephrasing it to say "The vent is in fault". However, I will point out that the names are pretty self explanatory, so you can probably get away with not adding any of these docstrings. That is the beauty of using expressive names, they document themselves.